spec/controllers/admin/reports/registered_adviser_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe Admin::Reports::RegisteredAdviserController, type: :controller do
+  describe '#filename' do
```

Comment:
IMHO, the only tests that you should worry about are the ones testing the public interface. Otherwise you may be coupling too tightly to implementation. Or make it the filename method to be public. Or even better, you can move the timestamp call to the renderer object and then your acceptance tests cover it.